packages/infrastructure/kube_nats_jetstream/nats-server.conf

```diff
@@ -0,0 +1,44 @@
+server_name: $NATS_SERVER_NAME
+listen: 0.0.0.0:4222
+http: 0.0.0.0:8222
+
+system_account: sys
+accounts: {
```

Comment:
This appears to be setting up a multi-tentant system. This seems unnecessary. Perhaps this is the issue?